common/src/OpenSpaceNet.cpp

```diff
@@ -459,31 +463,17 @@ void OpenSpaceNet::processSerial()
     skipLine();
     OSN_LOG(info) << "Reading image...";
 
-    unique_ptr<boost::progress_display> openProgress;
     if(!args_.quiet) {
-        openProgress = make_unique<boost::progress_display>(50);
+        pd_->start();
     }
 
-
     auto startTime = high_resolution_clock::now();
-    auto mat = GeoImage::readImage(*image_, bbox_, regionFilter_.get(), [&openProgress](float progress) -> bool {
-        size_t curProgress = (size_t)roundf(progress*50);
-        if(openProgress && openProgress->count() < curProgress) {
-            *openProgress += curProgress - openProgress->count();
-        }
-        return true;
+    auto mat = GeoImage::readImage(*image_, bbox_, regionFilter_.get(), [this](float progress) -> bool {
+        pd_->update(0, progress);
+        return !pd_->cancelled();
     });
 
     duration<double> duration = high_resolution_clock::now() - startTime;
-    OSN_LOG(info) << "Reading time " << duration.count() << " s";
-
-    skipLine();
-    OSN_LOG(info) << "Detecting features...";
```

Comment:
Without "Detecting features" it jumps right from "Reading image.." to a progress bar (that includes "Reading" and "Detecting".  It might be worth a log line that is something like "Processing..."

Currently:
```
Initializing the output feature set...

Reading image...

|  Reading  | Detecting |
+-----------+-----------+
|  100.0%   |  100.0%   |

Reading time 0.0123802 s
Detection time 0.256057 s

70 features detected.
```
Suggested something like:
```
Initializing the output feature set...
Loading image...
Processing...

|  Reading  | Detecting |
+-----------+-----------+
|  100.0%   |  100.0%   |

Reading time 0.0123802 s
Detection time 0.256057 s

70 features detected.
```